extmod/uasyncio/event.py

```diff
@@ -29,3 +33,24 @@ async def wait(self):
             core.cur_task.data = self.waiting
             yield
         return True
+
+
+_MP_STREAM_POLL = const(3)
```

Comment:
to eliminate the need for a `from micropython import const` I'd say just remove this line and hard-code the value with a comment below